use.py

```diff
@@ -0,0 +1,91 @@
+import os
```

Comment:
Could you put a comment about how to use this file and an example on what arguments it expects
`python use.py arg1 arg2 ...`

Reply:
If you pass --help as an argument, the other valid arguments with descriptions are listed. For our current uses no arguments should be given.